bin/setup.dart

```diff
@@ -53,7 +53,8 @@ void main() async {
     print('dependency on package:webcrypto is required');
     exit(1);
   }
-  final webcryptoRoot = packageConfigFile.uri.resolve(pkg['rootUri'] ?? '');
+  final webcryptoRoot =
+      packageConfigFile.uri.resolve(pkg['rootUri'] + '/' ?? '');
```

Comment:
Not required on my end, but if we want to go down this route, then we should make it conditional to avoid duplicate slashes.